### DISCUSSION TITLE
Migrated my project to Typescript, what bundler is supported by PixiJS?

### DISCUSSION THREAD

**Discussion by afarber** [pixijs/pixijs#9671] [Q&A] [answered]
Good evening, I have the following question please -

I have a multiplayer word game written initially in Javascript for PixiJS 4, then I kept updating it over the many years:

![Animation](https://github.com/pixijs/pixijs/assets/1884458/c390971c-8f92-4639-ba63-1e565f940d25)

(The backend is written in Java and is using Jetty WebSockets. It is built using Maven and a Macbook with Homebrew.)

Currently it uses PixiJS 7. I have noticed that PixiJS project itself has migrated to TypeScript.

Being a not very experienced frontend developer, I have followed a Udemy course and migrated my source code to TypeScript...

That has been the nice part of the process... but the not very nice part is that I need to bundle my Javascript code generated by TypeScript. I tried using Webpack (and another Udemy course on it), but it seems to be very bloated for my purposes.

So my question is if Webpack is the only bundler supported by PixiJS or if there maybe another, simpler bundler. Or maybe even just using the TypeScript compiler to bundle my generated JavaScript files to one? I don't need any additional features and for example do not really need minifying.

Thank you for any insights!

**Comment by bigtimebuddy** [comment, marked as answer]
Webpack, Rollup, Parcel are ones that I've used. I know some folks like Vite. It should work with all these.
